test/test_linalg.py

```diff
@@ -1895,6 +1895,7 @@ def test_cholesky_solve_batched_non_contiguous(self, device, dtype):
             self.assertEqual(x, x_exp)
 
     @slowTest
+    @onlyCPU  # GPU magma has illegal memory access. See https://github.com/pytorch/pytorch/issues/48996
```

Comment:
Use skipCUDAIf like this:

https://github.com/pytorch/pytorch/blob/d307601365c3b848072b8b8381208aedc1a0aca5/test/test_linalg.py#L246